src/main/java/code/microsystem/controller/PatientController.java

```diff
@@ -106,6 +106,14 @@ public ResponseEntity<List<Patient>> findByPnameAndAge(@RequestParam("pname") St
 						.body(Collections.singletonMap("error","Patient Id does not exists....."+id));
 			}
 		}
+     //find Patients Sorted By Fees.(Ascending and Descending order)
+     		@GetMapping("/findPatientsByFees")
+     		public ResponseEntity<List<Patient>> findPatientsByFees(@RequestParam("sort") String sortdirection) throws PatientNotFoundException{
```

Comment:
change name of variable ,because sort in may be keyword in java
EX: type or sortingtype